src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
```

Comment:
It seems that the import of 'java.util.Locale' is not used in this class. Remember to remove unused imports to keep the code clean.